Formula/libbiosig.rb

```diff
@@ -0,0 +1,31 @@
+class Libbiosig < Formula
+  desc "Biosig library"
+  homepage "https://biosig.sourceforge.io"
+  url "https://downloads.sourceforge.net/project/biosig/BioSig%20for%20C_C%2B%2B/src/biosig4c%2B%2B-1.9.5.src.tar.gz"
+  sha256 "20e72a5a07d1bf8baa649efe437b4d3ed99944f0e4dfc1fbe23bfbe4d9749ed5"
+
+  depends_on "gawk" => :build
+  depends_on "gnu-sed" => :build
+  depends_on "gnu-tar" => :build
+  depends_on "pkg-config" => :build
+  depends_on "dcmtk"
+  depends_on "suite-sparse"
+  depends_on "tinyxml" # if version == "1.9.5"
```

Comment:
Is this indicating that this dependency isn't needed in the next version?

Reply:
tinyxml was not needed in earlier versions, and is a new dependency in 1.9.5. It's not clear yet whether this will stay an external dependency, or whether tinyxml source will be included in the libbiosig sources. This is not decided yet. This is a reminder that this issue need to be revisited at the next release. If you do not mind, I'd like to keep this.